census/assets/dashboards/census_overview.json

```diff
@@ -0,0 +1,319 @@
+              "line_width": "normal"
+            },
+            "display_type": "bars"
+          }
+        ]
+      },
+      "layout": { "x": 4, "y": 0, "width": 4, "height": 2 }
+    },
+    {
+      "id": 6596770144114794,
+      "definition": {
+        "title": "Rows Processed",
+        "title_size": "16",
+        "title_align": "left",
+        "show_legend": true,
+        "legend_layout": "auto",
+        "legend_columns": ["avg", "min", "max", "value", "sum"],
+        "type": "timeseries",
+        "requests": [
+          {
+            "formulas": [{ "alias": "Rows Processed", "formula": "query1" }],
+            "response_format": "timeseries",
+            "queries": [
+              {
+                "query": "sum:census.syncs.rows_processed{*}.as_count()",
+                "data_source": "metrics",
+                "name": "query1"
+              }
+            ],
+            "style": {
+              "palette": "dog_classic",
+              "line_type": "solid",
+              "line_width": "normal"
+            },
+            "display_type": "line"
+          }
+        ]
+      },
+      "layout": { "x": 8, "y": 0, "width": 4, "height": 2 }
+    },
+    {
+      "id": 2702040426925268,
+      "definition": {
+        "title": "Successful Sync Runs",
+        "title_size": "16",
+        "title_align": "left",
+        "type": "query_value",
+        "requests": [
+          {
+            "formulas": [{ "formula": "query1" }],
+            "conditional_formats": [
+              { "comparator": ">", "palette": "white_on_green", "value": 0 }
+            ],
+            "response_format": "scalar",
+            "queries": [
+              {
+                "search": { "query": "status:ok census.syncs.completed" },
+                "data_source": "events",
+                "compute": { "aggregation": "count" },
+                "name": "query1",
+                "indexes": ["*"],
+                "group_by": []
+              }
+            ]
+          }
+        ],
+        "autoscale": true,
+        "precision": 2
+      },
+      "layout": { "x": 0, "y": 2, "width": 2, "height": 2 }
+    },
+    {
+      "id": 2594301627456280,
+      "definition": {
+        "title": "Failed Sync Runs",
+        "title_size": "16",
+        "title_align": "left",
+        "type": "query_value",
+        "requests": [
+          {
+            "formulas": [{ "formula": "query1" }],
+            "conditional_formats": [
+              { "comparator": ">", "palette": "white_on_red", "value": 0 }
+            ],
+            "response_format": "scalar",
+            "queries": [
+              {
+                "search": {
+                  "query": "census.syncs.completed status:emergency"
+                },
+                "data_source": "events",
+                "compute": { "aggregation": "count" },
+                "name": "query1",
+                "indexes": ["*"],
+                "group_by": []
```

Comment:
Please review this query. This is pulling in all events with the status 'Emergency' instead of Census specific events that have the status 'Emergency'.